cloud/pkg/devicecontroller/config/buffer.go

```diff
@@ -9,11 +9,32 @@ import (
 // UpdateDeviceStatusBuffer is the size of channel which save update device status message from edge
 var UpdateDeviceStatusBuffer int
 
+// DeviceEventBuffer is the size of channel which save device event from k8s
+var DeviceEventBuffer int
+
+// DeviceModelEventBuffer is the size of channel which save devicemodel event from k8s
+var DeviceModelEventBuffer int
+
 func init() {
 	if psb, err := config.CONFIG.GetValue("devicecontroller.update-device-status-buffer").ToInt(); err != nil {
 		UpdateDeviceStatusBuffer = constants.DefaultUpdateDeviceStatusBuffer
 	} else {
 		UpdateDeviceStatusBuffer = psb
 	}
 	log.LOGGER.Infof("Update device status buffer: %d", UpdateDeviceStatusBuffer)
+
+	if deb, err := config.CONFIG.GetValue("device-event-buffer").ToInt(); err != nil {
```

Comment:
perhaps we need to replace 'device-event-buffer' into 'devicecontroller.device-event-buffer'
The below config should be that too.

Reply:
ok